versionedController/v1/login/login.go

```diff
@@ -59,22 +59,28 @@ func (login *LoginController) Get(c *gin.Context) {
 	authType := c.Query("auth_type")
 	switch models.AuthType(authType) {
 	case models.GithubAuth:
-		{
-			if !controller.Server.Config.DisableGithubAuth {
-				githubURL := controller.Server.GithubConfig.AuthCodeURL(types.GithubState)
-				c.Redirect(http.StatusFound, githubURL)
-			} else {
-				c.JSON(http.StatusBadRequest, gin.H{
-					"error": "Authentication type not allowed",
-				})
-			}
+		if !controller.Server.Config.DisableGithubAuth {
+			githubURL := controller.Server.GithubConfig.AuthCodeURL(types.GithubState)
+			c.Redirect(http.StatusFound, githubURL)
+		} else {
+			c.JSON(http.StatusBadRequest, gin.H{
+				"error": "Authentication type not allowed",
+			})
 		}
-	default:
-		{
+	case models.GoogleAuth:
+		if !controller.Server.Config.DisableGoogleAuth {
+			googleURL := controller.Server.GoogleConfig.AuthCodeURL(types.GoogleState)
+			log.Info(googleURL)
```

Comment:
Do we need this?

Reply:
I've added this for debugging purposes, I'll remove it when I test everything works, as of now, the UI is getting redirected back to the main screen and the code isn't getting exchanged for the `access_token`.